requirements/requirements.txt

```diff
@@ -12,3 +12,4 @@ hydra-core>=1.0.4
 transformers>=4.0.1
 sentencepiece<1.0.0
 webdataset
+onnx-graphsurgeon
```

Comment:
@borisfom please make sure one can install this using regular `pip install` - in my case it failed, as it seems to require  nvidia-pyindex:

```
(nemo-env)tkornuta@tkornuta-mlt:~ $ pip install onnx-graphsurgeon
Collecting onnx-graphsurgeon
  Downloading https://files.pythonhosted.org/packages/a9/80/d0633ca3158f9712ba3b003e2046fe6f299fe53f9ba6e2a48dbea95c703b/onnx-graphsurgeon-0.0.1.dev4.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /Users/tkornuta/opt/anaconda3/envs/nemo-env/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-morv5knm/onnx-graphsurgeon/setup.py'"'"'; __file__='"'"'/private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-morv5knm/onnx-graphsurgeon/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-morv5knm/onnx-graphsurgeon/pip-egg-info
         cwd: /private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-morv5knm/onnx-graphsurgeon/
    Complete output (15 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-morv5knm/onnx-graphsurgeon/setup.py", line 150, in <module>
        raise RuntimeError(open("ERROR.txt", "r").read())
    RuntimeError:
    ###########################################################################################
    The package you are trying to install is only a placeholder project on PyPI.org repository.
    This package is hosted on NVIDIA Python Package Index.

    This package can be installed as:
    ```
    $ pip install nvidia-pyindex
    $ pip install onnx-graphsurgeon
    ```
    ###########################################################################################
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Reply:
I think I have fixed requirements.txt to do it properly - please try again! 
Note: stray 'pip install onnx-graphsurgeon' won't work, you need to either run pip install . in Nemo dir, or pip install -r requirements/requirements.txt

Reply:
@tkornuta-nvidia, can we resolve this conversation and merge?

Reply:
@borisfom I have tried both suggested methods on my MacBook and both failed, example:
```
    ERROR: Command errored out with exit status 1:
     command: /Users/tkornuta/opt/anaconda3/envs/nemo-env/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-8cin5y17/nvidia-pyindex/setup.py'"'"'; __file__='"'"'/private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-8cin5y17/nvidia-pyindex/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-8cin5y17/nvidia-pyindex/pip-egg-info
         cwd: /private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-8cin5y17/nvidia-pyindex/
    Complete output (6 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/83/1230013j3l33cd66ryccw4qh0000gp/T/pip-install-8cin5y17/nvidia-pyindex/setup.py", line 216, in <module>
        "supported.".format(sys.platform))
    OSError: Your operating system is not supported by `nvidia-pyindex`: darwin.
    Only Linux systems are supported.
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```
This is new and I am surprised by this - has someone decided to take the plug off from supporting nvidia-pypi for OSX??

@blisc Can you please "test" this PR/the new dependency (`onnx-graphsurgeon`) the way tested you EFF, i.e. try to install the it outside of the NVIDIA network? If this will work, both thumbs up to merge.